ui/narrative/methods/run_megahit/spec.json

```diff
@@ -2,7 +2,7 @@
 	"ver": "1.0.4",
```

Comment:
Can you bump the version number here?

Reply:
oops.  read too quickly and changed kbase.yml.  hold on...

Reply:
done